examples/Math_Test.eve

```diff
@@ -0,0 +1,273 @@
+# Math Module
```

Comment:
I think you can simply `git revert 5fcba7ff` and push.
That should do the trick.